Move clickhouse queries of death to public docs (#2509)
getsentry/snuba: docs/source/clickhouse/death_queries.rst

Clickhouse Queries Of Death
===========================

> The following queries have been shown to segfault clickhouse on 20.7 (which is what snuba is running). Do not run these queries in the tracing tool, unless you really want to take clickhouse down.

Reviewer: ```suggestion
The following queries have been shown to segfault clickhouse on 20.7 (which is the minimum Clickhouse version of Snuba). Do not run these queries in the tracing tool, unless you really want to take clickhouse down.
```

for our open source users

Reviewer: Also, did this segfault 20.3? Because I think that's the actual min version in the console

Author: I'm pretty sure we run 20.7, but also possibly 20.3?
